src/sid/simulate.py

```diff
@@ -59,7 +61,10 @@ def get_simulate_func(
         initial_states (pandas.DataFrame): See :ref:`states`. Cannot contain the column
             "date" because it is used internally.
         initial_infections (pandas.Series): Series with the same index as states with
-            initial infections.
+            initial infections. They are initial known cases. Sid will assume that
```

Comment:
I think as it is now, the `initial_infections` should be part of the `initial_conditions` dictionary.

Reply:
Make sense. It is not necessary if we start the simulation from a previous states DataFrame.